docs/scenarios_authoring/Spel.md

```diff
@@ -259,7 +263,15 @@ numeric types and between `String` and some useful logical value types. Some exa
 | `'USD'`                                  | String     | Currency               |
 | `'bf3bb3e0-b359-4e18-95dd-1d89c7dc5135'` | String     | UUID                   |
 
```

Comment:
I would add a link to [this section](https://nussknacker.io/documentation/docs/scenarios_authoring/Spel/#conversions-between-datetime-types) as some implicit conversions are enumerated here.

Reply:
Done